### PR TITLE
Add entity POST and PUT examples

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description

Add POST /entities and PUT entities/:entity examples.

Epic #1452 

### Review Instructions

We are also missing a POST /entities/:entity example, but when I tried this call in Postman, I got a "405 Method Not Allowed" response.

### CODE REVIEW

File: content/sensu-go/5.16/api/entities.md [hunk 1]
```diff
@@ -28,7 +28,10 @@ The `/entities` API endpoint provides HTTP GET access to [entity][1] data.
 The following example demonstrates a request to the `/entities` API endpoint, resulting in a JSON array that contains the [entity definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/entities -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/entities \
+"Authorization: Bearer $SENSU_TOKEN" \
```
Comment (rgeniesse) [inline content/sensu-go/5.16/api/entities.md]: should there be a `-H` at the start of this line?

File: content/sensu-go/5.16/api/entities.md [hunk 2]
```diff
@@ -213,7 +244,10 @@ The `/entities/:entity` API endpoint provides HTTP GET access to [entity data][1
 In the following example, querying the `/entities/:entity` API endpoint returns a JSON map that contains the requested [`:entity` definition][1] (in this example, for the `:entity` named `sensu-centos`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/entities/sensu-centos -H "Authorization: Bearer $SENSU_TOKEN"
+curl X GET \
```
Comment (rgeniesse) [inline content/sensu-go/5.16/api/entities.md]: Should the X have a dash in front of it?

File: content/sensu-go/5.16/api/entities.md [hunk 3]
```diff
@@ -357,6 +391,23 @@ output               | {{< highlight json >}}
...
+
+In the following example, an HTTP POST request is submitted to the `/entities/:entity` API endpoint to create the entity named `sensu-centos`.
+The request includes the updated entity definition in the request body and returns a successful `HTTP 201 Created` response.
+
+{{< highlight shell >}}
+curl -X POST \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+-H 'Content-Type: application/json' \
+-d '{
+PLACEHOLDER
```
Comment (rgeniesse) [inline content/sensu-go/5.16/api/entities.md]: Just calling out that this is still a placeholder.

**Review by rgeniesse — CHANGES_REQUESTED** [review]
I did not test the commands myself, but can do so once the inline comments are resolved.

File: content/sensu-go/5.16/api/entities.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/api/entities.md]: Yes, thank you

Comment (hillaryfraley) [inline content/sensu-go/5.16/api/entities.md]:
```suggestion
-H "Authorization: Bearer $SENSU_TOKEN" \
```

File: content/sensu-go/5.16/api/entities.md (hunk 2 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/api/entities.md]: yes again 😬

Comment (hillaryfraley) [inline content/sensu-go/5.16/api/entities.md]:
```suggestion
curl -X GET \
```

File: content/sensu-go/5.16/api/entities.md (hunk 3 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/api/entities.md]: yes -- I need someone to verify whether this call works. When I try it, I get a "405 Method Not Allowed" response.

Comment (rgeniesse) [reply content/sensu-go/5.16/api/entities.md]: ah ok, I'll take a closer look then.

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
just need to remove the last \ for the multi line commands

File: content/sensu-go/5.16/api/entities.md [hunk 4]
```diff
@@ -177,6 +180,34 @@ output         | {{< highlight shell >}}
...
 
+#### EXAMPLE {#entities-post-example}
+
+In the following example, an HTTP POST request is submitted to the `/entities` API endpoint to create a proxy entity named `sensu-centos`.
+The request includes the entity definition in the request body and returns a successful `HTTP 201 Created` response.
+
+{{< highlight shell >}}
+curl -X POST \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+-H 'Content-Type: application/json'
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/entities.md]:
```suggestion
-H 'Content-Type: application/json' \
```

File: content/sensu-go/5.16/api/entities.md [hunk 5]
```diff
@@ -357,6 +391,23 @@ output               | {{< highlight json >}}
...
 
+#### EXAMPLE {#entitiesentity-post-example}
+
+In the following example, an HTTP POST request is submitted to the `/entities/:entity` API endpoint to create the entity named `sensu-centos`.
+The request includes the updated entity definition in the request body and returns a successful `HTTP 201 Created` response.
+
+{{< highlight shell >}}
+curl -X POST \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+-H 'Content-Type: application/json'
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/entities.md]:
```suggestion
-H 'Content-Type: application/json' \
```

File: content/sensu-go/5.16/api/entities.md [hunk 6]
```diff
@@ -385,6 +436,36 @@ response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400
...
 
+#### EXAMPLE {#entitiesentity-put-example}
+
+In the following example, an HTTP PUT request is submitted to the `/entities/:entity` API endpoint to update the entity named `sensu-centos`.
+The request includes the updated entity definition in the request body and returns a successful `HTTP 201 Created` response.
+
+{{< highlight shell >}}
+curl -X PUT \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+-H 'Content-Type: application/json'
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/entities.md]:
```suggestion
-H 'Content-Type: application/json' \
```

**Comment by hillaryfraley** [comment]
I removed this placeholder so the PR can be merged. I opened the request to test this endpoint as https://github.com/sensu/sensu-docs/issues/2097.

#### EXAMPLE {#entitiesentity-post-example}

In the following example, an HTTP POST request is submitted to the `/entities/:entity` API endpoint to create the entity named `sensu-centos`.
The request includes the updated entity definition in the request body and returns a successful `HTTP 201 Created` response.

{{< highlight shell >}}
curl -X POST \
-H "Authorization: Bearer $SENSU_TOKEN" \
-H 'Content-Type: application/json' \
-d '{
PLACEHOLDER
}' \
http://127.0.0.1:8080/api/core/v2/namespaces/default/entities/sensu-centos

HTTP/1.1 201 Created
{{< /highlight >}}

**Pull request merged**